src/parser/ClassStatement.ts

```diff
@@ -327,6 +328,12 @@ export class ClassStatement implements Statement {
         result.push(`end function`);
         return result;
     }
+
+    walk(visitor: WalkVisitor, options?: WalkOptions) {
+        for (let i = 0; i < this.body.length; i++) {
```

Comment:
As I noted, `for i` isn't faster than `forEach` nowadays. Even slower in some benchmarks.

Reply:
`for i` is definitely faster than `forEach`. I just ran a few local benchmarks testing various loop implementations. See [this gist](https://gist.github.com/TwitchBronBron/16f3b20f76c66527169a61f5c2e1d0ce) for the test results. 

However, in this situation, we actually do need the loop index, because we pass it into the `walk` helper function.